processor/ratelimitprocessor/README.md

```diff
@@ -14,15 +14,29 @@ a in-memory rate limiter, or makes use of a [gubernator](https://github.com/gube
 | `throttle_behavior` | Processor behavior for when the rate limit is exceeded. Options are `error`, return an error immediately on throttle and does not send the event, and `delay`, delay the sending until it is no longer throttled. | Yes      | `error`    |
+| `throttle_interval` | Time interval for throttling. It has effects only when `type` is `gubernator`.                                                                                                                                    | No       | `1s`       |
 | `type`              | Type of rate limiter. Options are `local` or `gubernator`.                                                                                                                                                        | No       | `local`    |
+| `overrides`         | Allows customizing rate limiting parameters for specific metadata key-value pairs. Use this to apply different rate limits to different tenants, projects, or other entities identified by metadata. Each override is keyed by a metadata value and can specify custom `rate`, `burst`, and `throttle_interval` settings that take precedence over the global configuration for matching requests. | No       |            |
+
+### Overrides
+
+You can override one or more of the following fields:
+
+| Field               | Description                                                                                                                                                                                                       | Required | Default    |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|------------|
+| `rate`              | Bucket refill rate, in tokens per second.                                                                                                                                                                         | No       |            |
+| `burst`             | Maximum number of tokens that can be consumed.                                                                                                                                                                    | No       |            |
+| `throttle_interval` | Time interval for throttling. It has effect only when `type` is `gubernator`.                                                                                                                                     | No       |            |
+
+### Examples
 
 Example when using as a local rate limiter:
 
 ```yaml
 processors:
   ratelimiter:
     metadata_keys:
-    - x-elastic-project-id
+    - tenant-id
```

Comment:
Can you expand on why the example is changed to use `tenant-id`? Do you expect this to be the primary the use case rather than a per project throttling? (I assumed we would mostly consider rate limit overrides per project).